blt/src/Commands/HumsciCommand.php

```diff
@@ -247,4 +248,40 @@ public function updateAutoloader() {
     shell_exec("cd $root && composer dump-autoload");
   }
 
+  /**
+   * Create a new subtheme from the base subtheme.
+   *
+   * @command create-subtheme
+   */
+  public function createSubtheme() {
```

Comment:
This is neat but subthemes will need to exist in their own repo and/or branch at the root of the repository in order to be added by ACSF's system. https://docs.acquia.com/site-factory/theme/external/

Reply:
yeah i actually just talked with john about that. a little extra work in the future to break them out. but rather trival when the time comes